docs/rancher/csi-driver.md

```diff
@@ -40,65 +40,108 @@ When spinning up a Kubernetes cluster using Rancher RKE2 node driver, the Harves
 
 ![select-harvester-cloud-provider](/img/v1.2/rancher/rke2-cloud-provider.png)
 
-You can also deploy Harvester CSI Driver manually if you have already deployed the Harvester node driver manually. Perform the following steps to manually deploy the Harvester CSI Driver. 
+Suppose you would like to deploy the Harvester CSI driver without the cloud provider manually. You could create the Kubernetes cluster with RKE2 node driver with the default cloud provider (in v2.6, and the previous version is `None`).
```

Comment:
is `v2.6` referring to the Rancher version?

Reply:
Yes!
Thanks @guangbochen, @vickyhella for helping it more clear.